README.md

```diff
@@ -1,5 +1,5 @@
 # Sahaya Frontend
```

Comment:
We dont need Sahaya anymore

Reply:
update readme file has been taken care by another PR